app/models/communitarian/resolution.rb

```diff
@@ -15,7 +13,7 @@ def initialize(resolution_schedule)
     end
 
     def reopen_weekly_resolution!(original_post)
-      return if to_be_closed?(original_post)
+      return if to_be_closed?(original_post) || !reopen(original_post)
```

Comment:
What do you think if we simply will add additional check for `poll.close_at`? We can take today's date if poll has to be closed more than 1 day ago.
Example:
```
close_time = poll.close_at.today? ? poll.close_at : Time.current.midnight
poll.update!(close_at: resolution_schedule.next_close_time(close_time))
```